packages/@okta/vuepress-site/docs/reference/api/inline-hooks/index.md

```diff
@@ -478,12 +478,12 @@ A timeout of 3 seconds is enforced on all outbound requests, with one retry in t
 
 Note that this execution endpoint is not tied to any other functionality in Okta and should only be used for testing purposes.
 
-##### Response Parameters
+##### Response parameters
 
 Successful responses will return the full response returned by the external service, which should match the data contract for
 the given `inlineHookType` and version.
```

Comment:
Successful responses return the full response that is returned by the external service, which should match the data contract for the given `inlineHookType` and version.